jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/TypeHierarchyUtils.java

```diff
@@ -0,0 +1,99 @@
+package org.eclipse.lsp4jakarta.jdt.core;
```

Comment:
This class needs a header, see an example: https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/jax_rs/NoResourcePublicConstructorQuickFix.java#L1-L12